.gitignore

```diff
@@ -1,2 +1,3 @@
+ci/composer.lock
```

Comment:
Why? Why not use the latest psalm

Reply:
Because it's a moving target: since psalm does not yet build itself considering this library in its CI, we cannot really constantly have it as moving target, as the build breaks "semi-randomly".

It is possible to add a CI entry to run with un-locked dependencies, but the locked dependency on such an unstable dependency is very much needed for development to not be constantly derailed :)

Reply:
(yes, you can lock to a specific dev-master commit)